### DISCUSSION TITLE
VLC Media Player

### DISCUSSION THREAD

**Discussion by pog5** [catppuccin/catppuccin#2096] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

We could use the built in themes feature.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by bagelwaffle** [comment]
Hey,
I was looking at theming VLC the other day, and IIRC, VLC has to be themed their [skin editor](https://www.videolan.org/vlc/skineditor.html)

If I have free time soon, I might working on a draft....I don't know...maybe summer break :)

**Comment by Nyameliaaaa** [comment]
I am willing to write such a skin.

**Reply by ishandandekar** [reply]
Any updates on this?

**Comment by pog5** [comment]
We might have to agree on a layout since i dont think their skin editor allows for color theme-esqe skins

**Reply by 26lake** [reply]
well it should lmao

**Comment by Niandra8** [comment]
Any updates on it?

**Comment by yamcodes** [comment]
Any updates?

**Comment by pog5** [comment]
Using custom QT colors via KDE kinda does the trick for me

**Comment by jakeayy** [comment]
Any updates?

**Comment by y2w8** [comment]
Any updates?

**Comment by SHIMORA-6600X** [comment]
Any Updates ?

**Comment by pog5** [comment]
Any updates?

**Reply by SHIMORA-6600X** [reply]
Not yet
